yt_dlp/extractor/tudou.py

```diff
@@ -0,0 +1,64 @@
+from .common import InfoExtractor
+
+
+class TudouIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:play\.)?tudou\.com/v_show/(id_[a-zA-Z0-9_=.]+)'
```

Comment:
you did not have an `id` capture group, that is why `self._match_id()` was not working
```suggestion
    _VALID_URL = r'https?://(?:play\.)?tudou\.com/v_show/(?P<id>id_[\w_=.]+)'
```

Reply:
Updated